public/serviceWorker.js

```diff
@@ -1,6 +1,6 @@
 //////////////////////////////////////////
 // DIVBLOX GENERATED CODE - DO NOT MODIFY!
-const dxBuildTimeStamp = "14jul2022b";
+const dxBuildTimeStamp = "12Aug2022";
```

Comment:
I always like to put an "a" or "b" at the end of the date to allow for multiples on a single day